custom_components/better_thermostat/models/models.py

```diff
@@ -160,16 +160,16 @@ def convert_outbound_states(self, hvac_mode) -> Union[dict, None]:
 					_LOGGER.warning(
 						f"better_thermostat {self.name}: device config expects no system mode, while the device has one. Device system mode will be ignored"
 					)
-				hvac_mode = None
 				if hvac_mode == HVAC_MODE_OFF:
+					hvac_mode = None
 					_new_heating_setpoint = 5
 			
 			elif _has_system_mode is None and _system_mode is None:
-				hvac_mode = None
```

Comment:
If the TRV doesn't seem to have a system mode, we shouldn't try to send 'em one, shoudn't we?